app/helpers/storage_helper/textual_summary.rb

```diff
@@ -57,7 +57,7 @@ def textual_total_space
 
   def textual_uncommitted_space
     return nil if @record["total_space"].nil?
-    space = if @record["uncommitted"].blank? || @record["uncommitted"] == ""
+    space = if @record['uncommitted'].blank?
```

Comment:
We have about dozen of files that are consistent in regards to quotes (this one is not), so the consistency argument I cannot use for myself. I need to have different rule.

The rule: I change double quotes to single quotes, when touching an expression (not line).

I change to single quotes, because sooner or later I will find myself again in that file around and I will pity myself for not changing it in past.